### PR TITLE
Conseil 88: async db calls 2

### LINKED ISSUE #88: Conseil should not block on any database operations
At the moment, Conseil blocks indefinitely on database operations and this is a potential bottleneck. This should be changed to make all such operations non-blocking.

### PR DESCRIPTION
**Pull request by ivanopagano**
closes #88 

This second step includes missing changes to make all database operations return async results, combined there and then.
Only the main app should wait for results now.

### CODE REVIEW

File: src/main/scala/tech/cryptonomic/conseil/Lorre.scala [hunk 1]
```diff
@@ -1,22 +1,22 @@
 
 package tech.cryptonomic.conseil
 
+import akka.Done
 import com.typesafe.config.ConfigFactory
 import com.typesafe.scalalogging.LazyLogging
-import tech.cryptonomic.conseil.tezos._
+import tech.cryptonomic.conseil.tezos.{FeeOperations, TezosNodeInterface, TezosNodeOperator, TezosDatabaseOperations => TezosDb, _}
```
Comment (vishakh) [inline src/main/scala/tech/cryptonomic/conseil/Lorre.scala]: How come you use this alias here?

File: src/main/scala/tech/cryptonomic/conseil/Lorre.scala [hunk 2]
```diff
@@ -51,61 +52,64 @@ object Lorre extends App with LazyLogging {
...
           noOp
     } yield ()
+
     Await.ready(processing, atMost = Duration.Inf)
     logger.info("Taking a nap")
     Thread.sleep(sleepIntervalInSeconds * 1000)
     mainLoop(iteration + 1)
   }
 
+  sys.addShutdownHook{
```
Comment (vishakh) [inline src/main/scala/tech/cryptonomic/conseil/Lorre.scala]: Is this block in conflict with #154?

File: src/main/scala/tech/cryptonomic/conseil/tezos/Tables.scala [hunk 3]
```diff
@@ -204,6 +204,9 @@ trait Tables {
   /** Collection-like TableQuery object for table OperationGroups */
   lazy val OperationGroups = new TableQuery(tag => new OperationGroups(tag))
 
+  val operationGroupsByHash =
```
Comment (vishakh) [inline src/main/scala/tech/cryptonomic/conseil/tezos/Tables.scala]: This is a generated file produced using Slick via `GenSchema.scala`. We should move any custom logic out of here or make these changes in the database schema so this functionality is also also auto-generated.

File: src/main/scala/tech/cryptonomic/conseil/Lorre.scala (hunk 1 above)
Comment (ivanopagano) [reply src/main/scala/tech/cryptonomic/conseil/Lorre.scala]: I found it noisy at usage site

I'll revert if it's confusing

File: src/main/scala/tech/cryptonomic/conseil/Lorre.scala (hunk 2 above)
Comment (ivanopagano) [reply src/main/scala/tech/cryptonomic/conseil/Lorre.scala]: Yes sure, will need to properly merge.

I couldn't keep track anymore of where each change was made, so duplication crops out here and there.

File: src/main/scala/tech/cryptonomic/conseil/tezos/Tables.scala (hunk 3 above)
Comment (ivanopagano) [reply src/main/scala/tech/cryptonomic/conseil/tezos/Tables.scala]: Sorry, my bad, didn't notice, I'll take care

File: src/main/scala/tech/cryptonomic/conseil/Lorre.scala (hunk 1 above)
Comment (vishakh) [reply src/main/scala/tech/cryptonomic/conseil/Lorre.scala]: Generally, we try to avoid aliasing but we can see your point.

**Pull request merged**